x-pack/plugins/maps/public/classes/layers/create_region_map_layer_descriptor.ts

```diff
@@ -35,9 +34,13 @@ export function createAggDescriptor(metricAgg: string, metricFieldName?: string)
   });
   const aggType = aggTypeKey ? AGG_TYPE[aggTypeKey as keyof typeof AGG_TYPE] : undefined;
 
-  return aggType && metricFieldName
-    ? { type: aggType, field: metricFieldName }
-    : { type: AGG_TYPE.COUNT };
+  if (!aggType || aggType === AGG_TYPE.COUNT || !metricFieldName) {
```

Comment:
Same comment, is this needed since region map visualization does not support percentile aggregation?

Reply:
discussed offline, let's keep as is